packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -312,17 +332,33 @@ class AndroidCameraCameraX extends CameraPlatform {
   /// [cameraId] not used.
   @override
   Future<double> getMaxZoomLevel(int cameraId) async {
-    final ZoomState exposureState = await cameraInfo!.getZoomState();
-    return exposureState.maxZoomRatio;
+    final LiveData<ZoomState> liveZoomState = await cameraInfo!.getZoomState();
```

Comment:
Would it not be better to have a local `zoomState` variable that is updated by an `Observer` added to `liveZoomState`? Same for `getMinZoomLevel`.